### PR TITLE
TRUNK-6207: created platform 2.6 liquibase snapshot files

### PR DESCRIPTION
**Pull request by ManojLL**
## Description of what I changed

I added platform 2.6.x liquibase snapshot files

## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-6207

## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by ManojLL** [comment]
@dkayiwa could you please review this PR?

File: api/src/main/java/org/openmrs/liquibase/ChangeLogVersions.java [hunk 1]
```diff
@@ -27,14 +27,14 @@ public class ChangeLogVersions {
 	 * openmrs-core/api/src/main/resources/liquibase/snapshots/schema-only. If the actual change log
 	 * files and this list get out of sync, org.openmrs.liquibase.ChangeLogVersionsTest fails.
 	 */
-	private static final List<String> SNAPSHOT_VERSIONS = Arrays.asList("1.9.x", "2.1.x", "2.2.x", "2.3.x", "2.4.x", "2.5.x");
+	private static final List<String> SNAPSHOT_VERSIONS = Arrays.asList("1.9.x", "2.1.x", "2.2.x", "2.3.x", "2.4.x", "2.5.x","2.6.x");
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/liquibase/ChangeLogVersions.java:30]: Can we include space after the comma?

**Comment by dkayiwa** [comment]
@ManojLL can you take a look at the above build failure?

**Comment by ManojLL** [comment]
@dkayiwa I made some updates , Can you review them ?

**Comment by ManojLL** [comment]
@dkayiwa I fixed the build failure

**Comment by ManojLL** [comment]
@dkayiwa  could you please review this PR?

**Pull request merged**